snuba/datasets/processors/search_issues_processor.py

```diff
@@ -77,6 +98,13 @@ def ensure_uuid(value: str) -> str:
 class SearchIssuesMessageProcessor(DatasetMessageProcessor):
     FINGERPRINTS_HARD_LIMIT_SIZE = 100
 
+    PROMOTED_TAGS = {
```

Comment:
What does promoted mean in this context?

Reply:
These are basically 'special' tag values we lift directly from tags and place them in top-level columns. I basically took this pattern from how transactions was written. I presume this was done for performance - so instead of indexing into the tags array, we do some special processing when ingesting events and normalize it into its own specific column.